packages/admin-stories/src/docs/components/EditDialog/stories/useEditDialogApi.stories.tsx

```diff
@@ -0,0 +1,38 @@
+import { EditDialogApiContext, Field, FinalForm, FinalFormInput, useEditDialog, useEditDialogApi } from "@comet/admin";
+import { Button } from "@material-ui/core";
+import { storiesOf } from "@storybook/react";
+import * as React from "react";
+
+import { editDialogDecorator } from "../editDialog.decorator";
+
+const ChildComponentWithOpenButton: React.VoidFunctionComponent = () => {
+    const editDialogApi = useEditDialogApi();
+
+    return (
+        <Button onClick={() => editDialogApi?.openAddDialog()} variant="contained" color="primary">
+            Open Edit Dialog with useEditDialogApi()
+        </Button>
+    );
+};
```

Comment:
Maybe this component should be moved into the story, otherwise it's not visible when clicking "Show code" in storybook.